tests/Dockerfile.tests

```diff
@@ -3,7 +3,7 @@ COPY . .
 RUN make gobuild
 
 FROM quay.io/centos/centos:7
-ENV PACKAGES="jq git docker unzip wget"
+ENV PACKAGES="jq git docker unzip wget make"
```

Comment:
you can remove make from env packages if you move the docker command to the same place as the other docker container starts.

you can flow control errors the same way too